FoodTracker/Views/TableViewDataSource.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TableViewDataSource.swift
+//  FoodTracker
+//
+//  Created by BrainX IOS 4 on 2020-12-22.
+//
+
+import UIKit
+
+public class TableViewDataSource: NSObject, UITableViewDataSource, UITableViewDelegate {
```

Comment:
Whats the purpose of this class ? Where you are using it ?

Reply:
This class was removed from latest commit.